jetty-ee11/jetty-ee11-servlet/src/main/java/org/eclipse/jetty/ee11/servlet/ServletApiResponse.java

```diff
@@ -632,4 +656,132 @@ public String toString()
             return HttpCookie.toString(this);
         }
     }
+
+    public static class CrossContextInclude extends ServletApiResponse
```

Comment:
I think we need a comment about why this class is different from the `CrossContextDispatcher.IncludeResponse`